README.md

```diff
@@ -24,18 +24,31 @@ You can use your own computer as a server, but for other people to connect to yo
   [5]: http://en.wikipedia.org/wiki/Port_forwarding
 
 
-Installing
+Previous Requisites
 ------------------------------------------------------------------------
+Supported Node version is latest LTS 6.10.3. You can set the specific node version using:
+- [nvm](https://github.com/creationix/nvm) for Linux or Mac OS
+- [nvm-windows](https://github.com/coreybutler/nvm-windows) for Windows users
 
-    ./pokemon-showdown
+Then you can install this project's node version by running:
+```
+nvm install
+```
```

Comment:
Something of a motto that I have is "the job of a computer is to do automatable tasks for a human; if a human is doing automatable tasks for a computer, something is wrong".

So our project's usual way of doing this is to, in `./pokemon-showdown`, install `nvm` for the user if necessary, and leave Node alone if not.

In which case we might want to also remove `.nvmrc` and autogenerate it during the Node version detection phase, for a slightly cleaner root directory.